fakepkg

```diff
@@ -55,19 +56,15 @@ function fakebuild() {
 
 	# Assembling a list of all files belonging to the package
 	# The tar command is unable to handle very many arguments therefore create a file
-	pacman -Qlq $1 > ./package_file_list
-	echo ".PKGINFO" >> ./package_file_list
-	[[ -f ".INSTALL" ]] && echo ".INSTALL" >> ./package_file_list
-	[[ -f ".Changelog" ]] && echo ".Changelog" >> ./package_file_list
-	[[ -f ".MTREE" ]] && echo ".MTREE" >> ./package_file_list
+	pacman --dbpath "${ROOT_DIR}/var/lib/pacman" -Qlq $1 | sed 's%^/%%' > ./package_file_list
 
 	# Taring things together
 	if [[ $3 -ge "1" ]]; then
 		#XZ_OPT="-T 0" \
-		tar --ignore-failed-read --owner=0 --group=0 --no-recursion -c -a --force-local -f "${PKG_NAME}.pkg.tar.${PKG_EXT}" -T ./package_file_list
+		tar -C "${ROOT_DIR:-/}" --ignore-failed-read --owner=0 --group=0 --no-recursion -c -a --force-local -f "${PKG_NAME}.pkg.tar.${PKG_EXT}" -T ./package_file_list -C "$TMPDIR" .PKGINFO .MTREE .INSTALL .Changelog
```

Comment:
I doubt that specifying `-C` twice works as intended

Reply:
and yet..
```
$ /mkdir /tmp/{a,b}
$ touch /tmp/a/1 /tmp/b/2
$ tar -cvf /tmp/foo.tar -C /tmp/a 1 -C /tmp/b 2 
1
2
$ tar tvf /tmp/foo.tar 
-rw-r--r-- jose/users        0 2019-01-02 00:57 1
-rw-r--r-- jose/users        0 2019-01-02 00:57 2
```

Reply:
Thanks a bunch for the correction! I previously just took a look at the man page and though it would not be possible.

Either way, like I mentioned below, this change is somewhat independent from introducing a `-r` option and belongs into its own commit. However, to be completely honest, I would prefer to just amend the paths to `package_file_list`.

Reply:
This might've seen as a separate change but in fact it is not. With a default `ROOT_PATH` set to `/` `tar` has no troubles finding the files as `pacman -Qlq` output tells `tar` exactly where it should look for source files. This changes when you have to start elsewhere - `pacman` will give you `/usr/share/blabla` but you need to first change into your new `ROOT_PATH` - hence the `-C`. At the same time you need to keep `.PKGINFO`, `.MTREE` etc at the "top" of the archive.

Reply:
Right, agreed.